viam-orb-slam3/orbslam_server_v1.cc

```diff
@@ -146,6 +146,111 @@ ::grpc::Status SLAMServiceImpl::GetInternalStateStream(
     return grpc::Status::OK;
 }
 
+::grpc::Status SLAMServiceImpl::GetPosition(ServerContext *context,
+                                            const GetPositionRequest *request,
+                                            GetPositionResponse *response) {
```

Comment:
[nit] inconsistent formatting for multi-line argument function parameters. Low priority

Reply:
I tried changing this and the linter yelled at me, unsure why